chore(docs): asyncapi maintainership program docs update (#3541)
asyncapi/community: docs/maintainership-guide/amp-budget.md

weight: 190
---

Piloting an in-house mentorship program like the AsyncAPI Maintainership Program involves thoughtful financial planning. The program is designed to run in two modes depending on the Initiative's yearly financial standing: 

- **Funded Cohort:** Where the AsyncAPI Initiative allocates a portion of its annual budget to support the program.

- **Volunteer Cohort:** Where the program runs entirely on community goodwill with no stipends or honorariums.

Thanks to generous donations through our [Open Collective](https://opencollective.com/asyncapi), a funded cohort *may* be possible in a given year — but it is not guaranteed. Whether there's a budget or not, the program runs.

We've also created a dedicated [**Open Collective page**](https://opencollective.com/asyncapi/projects/asyncapi-mentorship) specifically for the Maintainership Program. This helps organizers easily manage and track program-specific expenses while keeping them loosely coupled from the broader Initiative's finances.

## Budget Overview

> Each year, before a new cohort is announced, the Initiative evaluates its finances to determine whether a funded cohort is feasible. If a budget is available, it is allocated to the program. If not, the cohort runs as a volunteer edition.

Reviewer: ```suggestion
Each year, before a new cohort is announced, the Initiative evaluates its finances to determine whether a funded cohort is feasible. If a budget is available, it is allocated to the program. If not, the cohort runs in a volunteer mode.
```
